2021-03-10 18:20:51,524 INFO    Thread-8  :1796 [wandb_setup.py:_flush():69] setting env: {'entity': '-my', 'project': 'dl_assignment1', 'root_dir': 'D:\\IITM\\deepLearning\\Assignments\\assignment1', 'sweep_id': 'xpd6qz04', 'run_id': 'ek2wk6gh', 'sweep_param_path': 'D:\\IITM\\deepLearning\\Assignments\\assignment1\\wandb\\sweep-xpd6qz04\\config-ek2wk6gh.yaml'}
2021-03-10 18:20:51,524 INFO    Thread-8  :1796 [wandb_setup.py:_flush():69] setting login settings: {}
2021-03-10 18:20:51,525 INFO    Thread-8  :1796 [wandb_init.py:_log_setup():319] Logging user logs to D:\IITM\deepLearning\Assignments\assignment1\wandb\run-20210310_182051-ek2wk6gh\logs\debug.log
2021-03-10 18:20:51,525 INFO    Thread-8  :1796 [wandb_init.py:_log_setup():320] Logging internal logs to D:\IITM\deepLearning\Assignments\assignment1\wandb\run-20210310_182051-ek2wk6gh\logs\debug-internal.log
2021-03-10 18:20:51,526 INFO    Thread-8  :1796 [wandb_init.py:init():352] calling init triggers
2021-03-10 18:20:51,526 INFO    Thread-8  :1796 [wandb_init.py:init():359] wandb.init called with sweep_config: {'alpha': 0, 'batch_size': 16, 'epochs': 10, 'hidden_Layer_AF': 'sigmoid', 'initializer': 'xavier', 'learning_rate': 0.001, 'loss_func': 'crossentropy', 'n_hidden_layers': 4, 'optimizer': 'nadam', 'size_hidden_layers': 64}
config: {'batch_size': 64, 'learning_rate': 0.001, 'epochs': 10, 'n_hidden_layers': 4, 'size_hidden_layers': 64, 'optimizer': 'adam'}
2021-03-10 18:20:51,527 INFO    Thread-8  :1796 [wandb_init.py:init():401] starting backend
2021-03-10 18:20:51,527 INFO    Thread-8  :1796 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=spawn, using: spawn
2021-03-10 18:20:51,537 INFO    Thread-8  :1796 [backend.py:ensure_launched():123] starting backend process...
2021-03-10 18:20:51,774 INFO    Thread-8  :1796 [backend.py:ensure_launched():128] started backend process with pid: 5496
2021-03-10 18:20:51,775 INFO    Thread-8  :1796 [wandb_init.py:init():406] backend started and connected
2021-03-10 18:20:51,781 INFO    Thread-8  :1796 [wandb_run.py:_config_callback():669] config_cb None None {'alpha': 0, 'batch_size': 16, 'epochs': 10, 'hidden_Layer_AF': 'sigmoid', 'initializer': 'xavier', 'learning_rate': 0.001, 'loss_func': 'crossentropy', 'n_hidden_layers': 4, 'optimizer': 'nadam', 'size_hidden_layers': 64}
2021-03-10 18:20:51,783 INFO    Thread-8  :1796 [wandb_init.py:init():446] updated telemetry
2021-03-10 18:20:51,784 INFO    Thread-8  :1796 [wandb_init.py:init():465] communicating current version
2021-03-10 18:20:54,653 INFO    Thread-8  :1796 [wandb_init.py:init():470] got version response upgrade_message: "wandb version 0.10.22 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-03-10 18:20:54,655 INFO    Thread-8  :1796 [wandb_init.py:init():478] communicating run to backend with 30 second timeout
2021-03-10 18:20:55,456 INFO    Thread-8  :1796 [wandb_init.py:init():503] starting run threads in backend
2021-03-10 18:21:00,476 INFO    Thread-8  :1796 [wandb_run.py:_console_start():1422] atexit reg
2021-03-10 18:21:00,613 INFO    Thread-8  :1796 [wandb_run.py:_redirect():1285] redirect: SettingsConsole.WRAP
2021-03-10 18:21:00,616 INFO    Thread-8  :1796 [wandb_run.py:_redirect():1320] Wrapping output streams.
2021-03-10 18:21:00,616 INFO    Thread-8  :1796 [wandb_run.py:_redirect():1336] Redirects installed.
2021-03-10 18:21:00,620 INFO    Thread-8  :1796 [wandb_init.py:init():527] run started, returning control to user process
2021-03-10 18:21:00,693 INFO    Thread-8  :1796 [wandb_setup.py:_flush():69] setting env: {'entity': '-my', 'project': 'dl_assignment1', 'root_dir': 'D:\\IITM\\deepLearning\\Assignments\\assignment1', 'sweep_id': 'xpd6qz04', 'run_id': 'ek2wk6gh', 'sweep_param_path': 'D:\\IITM\\deepLearning\\Assignments\\assignment1\\wandb\\sweep-xpd6qz04\\config-ek2wk6gh.yaml'}
2021-03-10 18:21:00,694 INFO    Thread-8  :1796 [wandb_setup.py:_flush():69] setting login settings: {}
2021-03-10 18:21:00,694 INFO    Thread-8  :1796 [wandb_init.py:_log_setup():319] Logging user logs to D:\IITM\deepLearning\Assignments\assignment1\wandb\run-20210310_182100-ek2wk6gh\logs\debug.log
2021-03-10 18:21:00,694 INFO    Thread-8  :1796 [wandb_init.py:_log_setup():320] Logging internal logs to D:\IITM\deepLearning\Assignments\assignment1\wandb\run-20210310_182100-ek2wk6gh\logs\debug-internal.log
2021-03-10 18:21:00,754 INFO    Thread-8  :1796 [wandb_init.py:init():352] calling init triggers
2021-03-10 18:21:00,754 INFO    Thread-8  :1796 [wandb_init.py:init():359] wandb.init called with sweep_config: {'alpha': 0, 'batch_size': 16, 'epochs': 10, 'hidden_Layer_AF': 'sigmoid', 'initializer': 'xavier', 'learning_rate': 0.001, 'loss_func': 'crossentropy', 'n_hidden_layers': 4, 'optimizer': 'nadam', 'size_hidden_layers': 64}
config: {}
2021-03-10 18:21:00,755 INFO    Thread-8  :1796 [wandb_init.py:init():398] wandb.init() called when a run is still active
2021-03-10 18:21:29,722 ERROR   Thread-7  :1796 [internal_api.py:execute():126] 429 response executing GraphQL.
2021-03-10 18:21:29,722 ERROR   Thread-7  :1796 [internal_api.py:execute():127] {"error":"rate limit exceeded"}

2021-03-10 18:23:14,591 ERROR   Thread-7  :1796 [internal_api.py:execute():126] 429 response executing GraphQL.
2021-03-10 18:23:14,591 ERROR   Thread-7  :1796 [internal_api.py:execute():127] {"error":"rate limit exceeded"}

